visitors.py

```diff
@@ -0,0 +1,15 @@
+import csv
+import io
+import urllib
+import zipfile
+
+import humanize
+import requests
+
+
+def get_number_of_visitors(language: str) -> str:
+    param = urllib.parse.urlencode({'filters': f'[["contains","event:page",["/{language}/"]]]'})
+    r = requests.get(f'https://plausible.io/docs.python.org/export?{param}', timeout=10)
+    with zipfile.ZipFile(io.BytesIO(r.content), 'r') as z, z.open('visitors.csv') as csv_file:
+        csv_reader = csv.DictReader(io.TextIOWrapper(csv_file))
+        return humanize.intword(sum(int(row["visitors"]) for row in csv_reader))
```

Comment:
I think it'd look better to show the total as a number ("195,463") rather than in words ("195.4 thousand"):
```suggestion
        return f"{sum(int(row["visitors"]) for row in csv_reader):,}"
```

<img width="1174" alt="image" src="https://github.com/user-attachments/assets/910633f1-007f-4c2e-8770-9e47c9638e7b">

<img width="1173" alt="image" src="https://github.com/user-attachments/assets/10cf96f9-6133-45d4-9a58-c26373abaf26">